utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
```

Comment:
Do you need regex?

Reply:
Was already imported in the code taken from Hackerrank.